modules/statics/src/coins.ts

```diff
@@ -7949,6 +7957,30 @@ export const coins = CoinMap.fromCoins([
     undefined,
     Networks.test.goerli
   ),
+  terc20(
+    '29c91204-eab5-499c-a623-1604c37b3848',
+    'tusdc',
+    'Goerli USDC',
+    18,
+    '0xef7ed2d5636c3dd9dc5e1d9a01fdb362d2927014',
+    UnderlyingAsset.TUSDC,
+    undefined,
+    undefined,
+    undefined,
+    Networks.test.goerli
+  ),
+  terc20(
+    '9aed2928-d976-4263-b80b-0e96d6fe6240',
+    'tusdt',
+    'Goerli USDT V2',
+    18,
+    '0x1697152c0e82b1a1e75ad2976f4d1e388ba65dff',
```

Comment:
Same for this one. Does this token have only 6 decimals? https://goerli.etherscan.io/token/0xc2c527c0cacf457746bd31b2a698fe89de2b6d49?a=0x1697152c0e82b1a1e75ad2976f4d1e388ba65dff